.gitlab-ci.yml

```diff
@@ -172,6 +173,18 @@ run_test_rpm-x64:
     # We also exclude python since centos 6 doesn't ship python and we run 'python' tests on the debian image.
     - inv -e test --race --profile --cpus 4 --build-exclude=systemd,python
 
+# run tests for eBPF code
+run_tests_ebpf:
+  stage: source_test
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/ebpf:latest
+  before_script:
+    - cd $SRC_PATH
+    - inv -e deps
+  tags: [ "runner:main", "size:large" ]
+  script:
+    # For now only check bpf bytes since we don't have a way to run eBPF tests without mounting a debugfs
```

Comment:
Is debugfs the problem? Or is it that we can't run the tests in an ebpf-enabled kernel?

Reply:
I'm not 100% sure on this but I think even if you are on a host that have an eBPF compatible kernel version you have to mount the `debugfs` in your container to make the tracer work